### ISSUE TITLE
getting zero-length error using basic.json example

### ISSUE THREAD

**Issue by alexciurea** [rmbrich/node-red-contrib-wdio#18]
```
Error: Invalid property expression: zero-length
    at createError (/opt/node_modules/@node-red/util/lib/util.js:180:13)
    at normalisePropertyExpression (/opt/node_modules/@node-red/util/lib/util.js:202:15)
    at getObjectProperty (/opt/node_modules/@node-red/util/lib/util.js:343:24)
    at Object.getMessageProperty (/opt/node_modules/@node-red/util/lib/util.js:316:12)
    at getTypeInputValue (/opt/node_modules/node-red-contrib-wdio/src/element-action.js:14:24)
    at elementAction._inputCallback (/opt/node_modules/node-red-contrib-wdio/src/element-action.js:54:27)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

Using: https://github.com/rmbrich/node-red-contrib-wdio/blob/master/examples/basic.json
Note: I also had to correct the send keys node from msg(hi) to str(hi).

The new session is targeting a local webdriver driver using selenium standalone chrome docker from dockerhub.
In the selenium webdriver page, I see the Chrome session created, and the screenshot shows it fills the search textbox with "hi", but the submit (click) fails with this kind of error.

Thanks,
-a

**Comment by alexciurea** [comment]
one workaround to continue the flow was to send Enter via sendKeys - for example: \uE007

**Comment by rmbrich** [comment]
I am unable to reproduce this. @cchintada-cng can you help with this?

**Comment by rmbrich** [comment]
I am able to reproduce this. It happens when using an element-action node, with the Send Keys action, where the Text to Send is set to a string. @cchintada-cng I believe this may be due to the change in https://github.com/rmbrich/node-red-contrib-wdio/pull/11 - can you take a look?

**Comment by alexciurea** [comment]
hi Rob, if I remember correctly, what you mentioned is another case that happened when using the browserless.io driver.

But in case of selenium, it fails at the element-actions, Click.
Few more details about the selenium container: https://hub.docker.com/r/selenium/standalone-chrome/
I tried with both debug and non debug versions & executed the container with these arguments: --shm-size 2g

From this error below, it looks like it expects a string that is not zero-length.
`at elementAction._inputCallback (/opt/node_modules/node-red-contrib-wdio/src/element-action.js:54:27)`

If I look at this file `element-action.js`, at line 54:
`let value = await getTypeInputValue(msg, config.object, config.sendKeys)`
In case of Click, sendKeys will end up being empty isn't it? (zero-length)
At least in the element-actions node, if user selects Click, then the sendKeys textbox is removed...

But I also tried different flow, on a different website with a login form, and the Click login button fails with this error "zero-length", but if I bypass this issue by using the sendKeys Enter key, I can login correctly, and then I can use element-actions Click to logout button **without a failure**... Maybe different types of buttons? not sure...

thanks
-a
